core/tests/drawing_tests.rs

```diff
@@ -23,6 +24,25 @@ fn parse_drawing_valid() {
         theme: None,
     };
     let drawing_bytes = serde_json::to_vec(&drawing).unwrap();
+    drawing::parse_drawing(&drawing_bytes).unwrap_err();
+}
+
+#[test]
+fn parse_drawing_valid() {
+    let drawing = Drawing {
+        scale: 0.0,
+        translation_x: 0.0,
+        translation_y: 0.0,
+        strokes: vec![Stroke {
+            points_x: vec![10f32, 50f32, 60f32],
+            points_y: vec![10f32, 50f32, 1000f32],
+            points_girth: vec![5f32, 7f32, 9f32],
```

Comment:
nit: I see we had this before, but unless the `f32` suffix is necessary here, `.0` is cleaner and easier to read imo and is more consistent.

Reply:
done!